client/my-sites/themes/theme-showcase.jsx

```diff
@@ -410,39 +461,57 @@ class ThemeShowcase extends Component {
 							select={ this.onTierSelect }
 						/>
 					) }
-					{ isLoggedIn && (
-						<SectionNav className="themes__section-nav" selectedText={ this.state.tabFilter.text }>
-							<NavTabs>
-								{ Object.values( this.tabFilters )
-									.sort( ( a, b ) => a.order - b.order )
-									.map(
-										( tabFilter ) =>
-											this.shouldShowTab( tabFilter.key ) && (
-												<NavItem
-													key={ tabFilter.key }
-													onClick={ () => this.onFilterClick( tabFilter ) }
-													selected={ tabFilter.key === this.state.tabFilter.key }
-													count={ this.notificationCount( tabFilter.key ) }
-												>
-													{ tabFilter.text }
-												</NavItem>
-											)
+					{ isLoggedIn &&
+						( isNewSearchAndFilter ? (
+							<div className="theme__filters">
+								<ThemesToolbarGroup
+									items={ Object.values( this.tabFilters ).filter( ( tabFilter ) =>
+										this.shouldShowTab( tabFilter.key )
 									) }
-							</NavTabs>
-						</SectionNav>
-					) }
+									selectedKey={ this.state.tabFilter.key }
+									onSelect={ ( key ) =>
+										this.onFilterClick(
+											Object.values( this.tabFilters ).find(
+												( tabFilter ) => tabFilter.key === key
+											)
+										)
+									}
+								/>
+								{ premiumThemesEnabled && ! isMultisite && (
+									<SimplifiedSegmentedControl
```

Comment:
Do we need to show it on the new version? I cannot see it on the Figma 🤔

Reply:
The spec can be found here: 0EogtxvlmxMGcnEEaMVECv-fi-1492%3A52007&t=K1fZL0gHpcnkIoOG-0
The PT also has a screenshot of the design: pbxlJb-2V0-p2